app/backend/src/createusers.py

```diff
@@ -0,0 +1,89 @@
+import csv
+import os
+import sys
+import math
+import random
+import logging
+import faker
+import multiprocessing
+from couchers.models import (
+    User,
+)
+from couchers import config
+from couchers.crypto import hash_password
+from couchers.servicers.api import hostingstatus2sql
+from couchers.utils import create_coordinate
+from couchers.db import session_scope
+
+fake = faker.Faker()
+
+logger = logging.getLogger(__name__)
+
+cities = []
+countries = set()
+world_population = 0
+
+# data from https://simplemaps.com/data/world-cities
+file_name = "src/data/worldcities.csv"
+with open(file_name, "r") as file:
+    reader = csv.reader(file)
+    try:
+        reader.__next__()
+        for row in reader:
+            if row[9] != "":
+                cities.append(row)
+                world_population += int(float(row[9]))
+                countries.add(row[4])
+    except csv.Error as e:
+        sys.exit("file {}, line {}: {}".format(file_name, reader.line_num, e))
+
+config.check_config()
```

Comment:
Remove this line; it requires a ton of unnecessary environment variables to be set; only DATABASE_CONNECTION_STRING is used.